oak_functions/loader/src/testing.rs

```diff
@@ -23,23 +23,24 @@ use crate::{
 };
 
 use log::Level;
-use oak_functions_abi::proto::OakStatus;
+use oak_functions_abi::{proto::OakStatus, ExtensionHandle};
 use serde::{Deserialize, Serialize};
 use wasmi::ValueType;
 
 /// Host function name for testing.
-const TESTING_ABI_FUNCTION_NAME: &str = "invoke";
+const TESTING_ABI_FUNCTION_NAME: &str = "testing";
 
 impl OakApiNativeExtension for TestingExtension<Logger> {
     fn invoke(
         &mut self,
         wasm_state: &mut crate::server::WasmState,
         args: wasmi::RuntimeArgs,
     ) -> Result<Result<(), oak_functions_abi::proto::OakStatus>, wasmi::Trap> {
-        let request_ptr: AbiPointer = args.nth_checked(0)?;
-        let request_len: AbiPointerOffset = args.nth_checked(1)?;
-        let response_ptr_ptr: AbiPointer = args.nth_checked(2)?;
-        let response_len_ptr: AbiPointer = args.nth_checked(3)?;
+        // We still read the args after the ExtensionHandle at position 0.
```

Comment:
Maybe add `let _handle = args.nth_checked(0);` for consistency and future proof.

Reply:
Done.